content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -157,6 +157,34 @@ jobs:
 ```
 {% endraw %}
 
+{% if actions-inherit-secrets-reusable-workflows %}
+
+#### `on.workflow_call.secrets.inherit`
+
+Use the `inherit` keyword to pass all secrets the calling workflow has access to, to the called workflow. This includes all organization, repository and environment secrets. The keyword can be used to pass secrets across organizations within the same enterprise. If there are duplicate names between the repository and organization, the repository secret will be used.
```

Comment:
I don't think it is necessary to mention this here, because it is already existing behavior; if there is a repo secret and org secret with the same name in a non-reusable workflow scenario, repo secret will be used.
```suggestion
Use the `inherit` keyword to pass all secrets the calling workflow has access to, to the called workflow. This includes all organization, repository and environment secrets. The keyword can be used to pass secrets across organizations within the same enterprise.
```

Reply:
100%